launcher/common/xml_manager_actions.py

```diff
@@ -0,0 +1,282 @@
+# ------------------------------------------------------------------------------
```

Comment:
Only a minor thing:
I would understand the name `actions_xml_manager.py` better.

I know that this might be a personal preference (naming convention of inheritance). So not an important change

Reply:
Totally agree (again!) , they were called in such way, I changed just to have them together based on inheritance as you mention it.